privilege-escalation/activities/writable/README.md

```diff
@@ -0,0 +1,33 @@
+# Name: Web: Privilege Escalation: Writable
+
+## Vulnerability
+
+Writable ```/etc/passwd``` file.
+
+## Exploit
+
+Connect with ssh to user ```jack:YvFWPeC7sTWJdaYQ```, port 2022.
+
+```ssh jack@<IP> -p 2022```
+
+If you look at the ```/etc/passwd``` file we can see that our user account has read/write access.
+
+```ls -la /etc/passwd```
+
+What we can do is to generate a new password using one of the following commands:
+
```

Comment:
Why did you remove the command with openSSL?
What I wanted to say in the last review was just to add some words that those commands do the same thing.